support-frontend/assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -87,4 +87,25 @@ export const tests: Tests = {
     seed: 10,
     optimizeId: '-2DJ0JTsSVaskWwTm9Je4A',
   },
+  comparisonTableTest: {
```

Comment:
Might be worth deactivating the events test as well in this PR?

Reply:
Good suggestion. I've put the audience size to zero for that test and to 1 for this one. I'll create a card to remove the events test.